service/worker/deletenamespace/deleteexecutions/workflow.go

```diff
@@ -186,6 +187,7 @@ func DeleteExecutionsWorkflow(ctx workflow.Context, params DeleteExecutionsParam
 	params.PreviousSuccessCount = result.SuccessCount
 	params.PreviousErrorCount = result.ErrorCount
 	params.ContinueAsNewCount++
+	params.NextPageToken = nextPageToken
 
 	logger.Info("There are more workflows to delete. Continuing workflow as new.", tag.WorkflowType(WorkflowName), tag.WorkflowNamespace(params.Namespace.String()), tag.DeletedExecutionsCount(result.SuccessCount), tag.DeletedExecutionsErrorCount(result.ErrorCount), tag.Counter(params.ContinueAsNewCount))
```

Comment:
nit: I think workflow execution history is good enough so this logging is not really needed.